### PR TITLE
Improve API Docs and add Package.MD file

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
$Subject

### CODE REVIEW

File: os-ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Module Overview
+
+This module provides APIs to retrieve information about the os and the current users of the os.
```
Comment (daneshk) [inline os-ballerina/Module.md:3]:
```suggestion
This module provides APIs to retrieve information about the OS and the current users of the OS.
```

File: os-ballerina/Package.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
-## Package overview
+## Package Overview
 
-This package provides functions to retrieve information about the operating system and the current users of the operating system.
+The OS library is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
 
-For information on the operations, which you can perform with the os package, see the below **Functions**.
+This package provides functions to retrieve information about the os and the current users of the os.
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).
```
Comment (daneshk) [inline os-ballerina/Package.md:7]:
```suggestion
```

File: os-ballerina/Module.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Module Overview
```
Comment (daneshk) [inline os-ballerina/Module.md:1]:
```suggestion
## Overview
```

File: os-ballerina/Module.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+## Overview
+
+This module provides APIs to retrieve information about the environment variables and the current user of the Operating System.
```
Comment (praneesha) [inline os-ballerina/Module.md:3]:
```suggestion
This module provides APIs to retrieve information about the environment variables and the current users of the Operating System.
```

File: os-ballerina/Module.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+The APIs are platform-independent, i.e. uniform across all operating systems.
```
Comment (praneesha) [inline os-ballerina/Module.md:5]:
```suggestion
The APIs are platform-independent, i.e., uniform across all operating systems.
```

File: os-ballerina/Module.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+For information on the operations, which you can perform with the os module, see the below **Functions**.
```
Comment (praneesha) [inline os-ballerina/Module.md:7]:
```suggestion
For information on the operations, which you can perform with the `os` module, see the below **Functions**.
```

File: README.md [hunk 7]
```diff
@@ -6,8 +6,8 @@ Ballerina OS Package
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/os.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fos)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-os/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-os)
-  
-The OS package is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+The OS library is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```
Comment (praneesha) [inline README.md:10]:
```suggestion
The `os` library is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

File: os-ballerina/Package.md [hunk 8]
```diff
@@ -1,5 +1,16 @@
-## Package overview
+## Package Overview
 
-This package provides functions to retrieve information about the operating system and the current users of the operating system.
+The OS library is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
 
-For information on the operations, which you can perform with the os package, see the below **Functions**.
+This package provides functions to retrieve information about the os and the current users of the os.
```
Comment (praneesha) [inline os-ballerina/Package.md:5]:
```suggestion
This package provides functions to retrieve information about the OS and the current users of it.
```

File: os-ballerina/Package.md [hunk 9]
```diff
@@ -1,5 +1,16 @@
...
+
+
+## Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```
Comment (praneesha) [inline os-ballerina/Package.md:10]:
```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

File: README.md [hunk 10]
```diff
@@ -6,8 +6,8 @@ Ballerina OS Package
...
 
 This package provides functions to retrieve information about the os and the current users of the os.
```
Comment (praneesha) [inline README.md:12]:
```suggestion
This package provides functions to retrieve information about the oS and the current users of it.
```

Comment (MadhukaHarith92) [reply README.md:12]: Shall we change `oS` -> `OS`

**Review by praneesha — APPROVED** [review]

File: README.md (hunk 10 above)
Comment (daneshk) [reply README.md:12]: Let's change to OS

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**